nixos/tests/common/acme/server/default.nix

```diff
@@ -54,11 +54,6 @@ let
   testCerts = import ./snakeoil-certs.nix;
   domain = testCerts.domain;
 
-  resolver = let
-    message = "You need to define a resolver for the acme test module.";
```

Comment:
What was the reason behind removing this?

Reply:
It's unused so has no effect.

Reply:
Ah, ok then 😅

Reply:
You should setup a LSP it'll highlight this stuff :)